simulator/src/test/java/com/hedera/block/simulator/config/TestConfigBuilder.java

```diff
@@ -0,0 +1,242 @@
+/*
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.block.simulator.config;
+
+import com.swirlds.common.config.singleton.ConfigurationHolder;
+import com.swirlds.common.threading.locks.AutoClosableLock;
+import com.swirlds.common.threading.locks.Locks;
+import com.swirlds.common.threading.locks.locked.Locked;
+import com.swirlds.config.api.ConfigData;
+import com.swirlds.config.api.Configuration;
+import com.swirlds.config.api.ConfigurationBuilder;
+import com.swirlds.config.api.converter.ConfigConverter;
+import com.swirlds.config.api.source.ConfigSource;
+import com.swirlds.config.api.validation.ConfigValidator;
+import com.swirlds.config.extensions.sources.SimpleConfigSource;
+import edu.umd.cs.findbugs.annotations.NonNull;
+import edu.umd.cs.findbugs.annotations.Nullable;
+import java.util.Objects;
+
+/**
+ * Helper for use the config in test and change the config for specific tests. Instance can be used
+ * per class or per test.
+ */
+public class TestConfigBuilder {
+
+    private final AutoClosableLock configLock = Locks.createAutoLock();
+
+    private Configuration configuration = null;
```

Comment:
Member variables are `null` by default. You could reduce this to: `private Configuration configuration;`

Reply:
part of the copy/paste, but no harm in simplifying it.

✅  Done!